envoy/api/v2/core/envoy_http_target.proto

```diff
@@ -0,0 +1,58 @@
+// [#protodoc-title: Protocol options]
+
+syntax = "proto3";
+
+package envoy.api.v2.core;
+
+import "validate/validate.proto";
+
+// When Envoy makes a HTTP call, two values are needed to specify the HTTP
+// target; "uri" for the HTTP URI and "cluster" for the cluster name in the
+// Envoy config.
+// [#not-implemented-hide:]
+message EnvoyHttpTarget {
+  // The HTTP server URI. It should be a full FQDN with protocol, host and path.
+  //
+  // Example:
+  //
+  // .. code-block:: yaml
+  //
+  //    uri: https://www.googleapis.com/oauth2/v1/certs
+  //
+  string uri = 1 [(validate.rules).string.min_bytes = 1];
+
+  // A cluster must be created in the Envoy "cluster_manager" config section.
+  // This field specifies the cluster name.
+  //
+  // Example:  a jwks cluster in "cluster_manager" section of Envoy config.
+  //
+  // .. code-block:: json
+  //
+  //    {
+  //      "name": "jwt.www.googleapis.com|443",
+  //      "connect_timeout_ms": 1000,
+  //      "type": "strict_dns",
+  //      "lb_type": "round_robin",
+  //         "hosts": [
+  //            {
+  //              "url": "tcp://www.googleapis.com:443"
+  //            }
+  //          ],
+  //      "ssl_context": {},
+  //      "circuit_breakers": {
+  //         "default": {
+  //           "max_pending_requests": 10000,
+  //           "max_requests": 10000
+  //         }
+  //      }
+  //    }
+  //
+  // Example:  uri and uri_cluster:
+  //
+  // .. code-block:: yaml
+  //
+  //    uri: https://www.googleapis.com/oauth2/v1/certs
+  //    uri_cluster: jwt.www.googleapis.com|443
+  //
+  string uri_cluster = 2 [(validate.rules).string.min_bytes = 1];
```

Comment:
We will support generic outbound calls in the future, so this won't always be necessary. I would suggest potentially breaking this down into sub-types that are selected via a oneof. For example, the only type specified right now might be a `StaticClusterCall` or something like that?

Reply:
Done